pkg/integration/tests/patch_building/apply_in_reverse_with_conflict.go

```diff
@@ -39,11 +39,15 @@ var ApplyInReverseWithConflict = NewIntegrationTest(NewIntegrationTestArgs{
 			).
 			// Add both files to the patch; the first will conflict, the second won't
 			PressPrimaryAction().
+			Tap(func() {
```

Comment:
I'm adding an assertion between pressing space and selecting the next item so that the test isn't going too fast for the code to keep up. The code _should_ prevent an insanely fast user from tripping up the code, but at the moment it is failing to do that.